diff --git a/common.gypi b/common.gypi
index 7727dfc..9c9b915 100644
--- a/common.gypi
+++ b/common.gypi
@@ -651,7 +651,7 @@
           'GCC_ENABLE_PASCAL_STRINGS': 'NO',        # No -mpascal-strings
           'GCC_STRICT_ALIASING': 'NO',              # -fno-strict-aliasing
           'PREBINDING': 'NO',                       # No -Wl,-prebind
-          'MACOSX_DEPLOYMENT_TARGET': '13.5',       # -mmacosx-version-min=13.5
+          'MACOSX_DEPLOYMENT_TARGET': '12.1',       # -mmacosx-version-min=12.1
           'USE_HEADERMAP': 'NO',
           'WARNING_CFLAGS': [
             '-Wall',
